.teamcity/build.sh

```diff
@@ -33,17 +33,48 @@ function run_example_test {
         cp parl/tests/gym.py examples/${exp}/
     done
```

Comment:
It's suggested to replace line 33 by `sed -i '/paddlepaddle/d' ./examples/{exp}/requirements.txt` and `sed -i '/parl/d' ./examples/{exp}/requirements.txt`